engine/lib/configuration.php

```diff
@@ -275,6 +275,33 @@ function _elgg_load_site_config() {
 }
 
 /**
+ * Set up CONFIG->cookies. (this is for unit testing)
+ *
+ * @see phpunit/bootstrap.php
+ *
+ * @param stdClass $CONFIG Elgg's config object
+ * @access private
+ */
+function _elgg_configure_cookies($CONFIG) {
+	// set cookie values for session and remember me
+	if (!isset($CONFIG->cookies)) {
```

Comment:
Agreed. I went to refactor this and noticed it array merges the values into possibly existing arrays in $CONFIG. Reworking this is a can of worms not worth my time at the moment.